chart/app-templates/crawler.yaml

```diff
@@ -61,7 +61,7 @@ spec:
   volumes:
     - name: crawl-config
       configMap:
-        name: crawl-config-{{ cid }}
+        name: crawl-config-{{ id }}
```

Comment:
Nice catch

Reply:
This is actually part of the change - before the configmaps were per crawl config(crawl workflow) and were named with the crawl config id (cid), now they are per crawl, so use the crawl id (just id here). Maybe its still too confusing :)

Reply:
ahhh right, no I think that makes sense!